packages/chart/src/StatChart.ts

```diff
@@ -0,0 +1,213 @@
+import { format as d3Format, HTMLWidget, Palette } from "@hpcc-js/common";
+import { QuartileCandlestick } from "./QuartileCandlestick";
+import { Scatter } from "./Scatter";
+
+const rainbow = Palette.rainbow("Blues");
+const palette = Palette.ordinal("Quartile", [rainbow(100, 0, 100), rainbow(50, 0, 100), rainbow(50, 0, 100), rainbow(75, 0, 100)]);
+palette("Std. Dev.");
+palette("MinMax");
+palette("25%");
+palette("50%");
+
+type QuertilesType = [number, number, number, number, number];
+type Mode = "min_max" | "25_75" | "normal";
+type Tick = { label: string, value: number };
+type Ticks = Tick[];
+type AxisTick = { label: string, value: string };
+type AxisTicks = AxisTick[];
+
+const CANDLE_HEIGHT = 20;   // Pixels
+const DOMAIN_PADDING = 10;  // Percentage
+
+class BellCurve extends Scatter {
+}
```

Comment:
Empty class not needed (it used to have some additional methods, which have recently been added to Scatter).